src/main/java/uk/gov/hmcts/reform/juddata/scheduler/JrdLeafDataLoadStarter.java

```diff
@@ -10,10 +10,10 @@
 import org.apache.camel.ProducerTemplate;
```

Comment:
could you please append schedular to the name of the class to make uniformity across the package?

Reply:
this will be changed in sooner next spring as all stuff gets replace with spring tasklet kubectl.

Reply:
I mean print ex object instead of ex.message()